client/components/Player.jsx

```diff
@@ -133,7 +102,8 @@ class Player extends React.Component {
   }
 
```

Comment:
the handlebarhover and exit seem to have really similar lines. suggestion would be to combine it into one function that can handle both

Reply:
I like this! It's a good idea! I'm implementing it